### PR TITLE
feat(integrations): add support for listmonk

### PR DESCRIPTION
**Pull request by samuelandert**
## Describe your changes
I integrated basic proxy support for listmonk. 

## Checklist before requesting a review (skip if just adding/editing APIs & templates)
- [ ] I added tests, otherwise the reason is: 
- [ ] I added observability, otherwise the reason is:
- [ ] I added analytics, otherwise the reason is:

### CODE REVIEW

File: docs-v2/integrations/all/listmonk.mdx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+sidebarTitle: Listmonk
+---
+
+API configurations: [`listmonk`](https://nango.dev/providers.yaml)
+
+## Features
+
+| Features                                                             | Status                           |
+| -------------------------------------------------------------------- | -------------------------------- |
+| [Auth (OAuth)](/integrate/guides/authorize-an-api)                   | ✅                               |
```
Comment (hassan254-prog) [inline docs-v2/integrations/all/listmonk.mdx]: Update this line to `[Auth (Basic)]`

File: packages/shared/providers.yaml [hunk 2]
```diff
@@ -1545,6 +1545,15 @@ luma:
             method: GET
             endpoint: /public/v1/user/get-self
     docs: https://docs.nango.dev/integrations/all/luma
+listmonk:
+    categories:
+        - marketing
+    auth_mode: BASIC
+    proxy:
+        base_url: https://${connectionConfig.domain}/api
+        headers:
```
Comment (hassan254-prog) [inline packages/shared/providers.yaml:1586]: With `API_KEY` and `BASIC` auth_modes, it is advisable to test your credentials using the (cf. `proxy.verification` field). Kindly look at the following [example](https://github.com/NangoHQ/nango/pull/2410/files#diff-ea826d3a45056e71af1c312cb0757f978c999d0b4d94afa0e6766afb8226f5ef)

File: packages/webapp/public/images/template-logos/listmonk.svg
Comment (hassan254-prog) [inline packages/webapp/public/images/template-logos/listmonk.svg:1]: Sharing this [svg](https://codefile.io/f/pWZDROW24q) file with the correct dimensions of the logo.

**Comment by hassan254-prog** [comment]
Hi @samuelandert ,thanks a lot for your contribution, just minor comments otherwise lgtm 💯

File: docs-v2/integrations/all/listmonk.mdx [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+<Tip>We can implement missing features in &lt;48h, just ask for it in the [community](https://nango.dev/slack).</Tip>
+
+## Getting started
+
+-   [Listmonk REST API docs](https://listmonk.app/docs/apis/apis/)
+
+<Tip>Need help getting started? Get help in the [community](https://nango.dev/slack).</Tip>
+
+## API gotchas
```
Comment (bodinsamuel) [inline docs-v2/integrations/all/listmonk.mdx:26]:
```suggestion
## API gotchas

_None yet, add yours!_

<Note>Add Getting Started links and Gotchas by [editing this page](https://github.com/nangohq/nango/tree/master/docs-v2/integrations/all/listmonk.mdx)</Note>

```

File: packages/shared/providers.yaml [hunk 4]
```diff
@@ -1545,6 +1545,15 @@ luma:
...
+            'Content-Type': 'application/json'
```
Comment (bodinsamuel) [inline packages/shared/providers.yaml]:
```suggestion
            content-type: application/json
```

**Review by bodinsamuel — APPROVED** [review]
Thanks for your contribution, Minor suggestions 👌🏻

File: packages/shared/providers.yaml (hunk 4 above)
Comment (samuelandert) [reply packages/shared/providers.yaml]: Should i omit the header as hassan suggested or use samuels suggestion?

Comment (hassan254-prog) [reply packages/shared/providers.yaml]: @samuelandert, please use Samuel's suggestion.

File: packages/webapp/public/images/template-logos/listmonk.svg
Comment (samuelandert) [reply packages/webapp/public/images/template-logos/listmonk.svg:1]: I am trying your svg but it is still not showing up in the UI

**Comment by samuelandert** [comment]
I think everything but the svg, which is still not showing is good to go.

**Review by TBonnin — APPROVED** [review]

**Comment by samuelandert** [comment]
Nice, thanks for merging this into master.
How long does it usually take until its available inside the nango.dev UI dashboard?

**Comment by bodinsamuel** [comment]
We usually deploy two or three times a week, probably going to happen today. I'll ping you when it's live 👌🏻

**Comment by samuelandert** [comment]
Lovely, thanks for the fast support and all the help. Great work!

**Comment by bodinsamuel** [comment]
Available in prod 🚀

**Pull request merged**